machine_learning_hep/submission/default_histomass.yml

```diff
@@ -0,0 +1,84 @@
+case: XXXX # used to find the database file unless specified explicitly as do_entire_analysis -d database_analysis
+download:
+  alice:
+    activate: false
+conversion: # pkl
+  mc:
+    activate: false
+  data:
+    activate: false
+skimming: # pkl_skimmed (pklsk), pkl_evtcounter_all
+  mc:
+    activate: false
+  data:
+    activate: false
+merging: # pkl_skimmed_merge_for_ml (pklskml)
+  mc:
+    activate: false
+  data:
+    activate: false
+mergingperiods: # pkl_skimmed_merge_for_ml_all
+  mc:
+    activate: false
+  data:
+    activate: false
+
+ml_study: # mlout, mlplot
+  activate: false
+  dopca: false
+  docorrelation: false
+  dotraining: false
+  dotesting: false
+  doapplytodatamc: false
+  docrossvalidation: false
+  dolearningcurve: false
+  doroc: false
+  doroctraintest: false
+  doplotdistr: false
+  doimportance: false
+  dogridsearch: false
+  doboundary: false
+  doefficiency: false
+  dosignifopt: false
+  doscancuts: false
+
+mlapplication:
+  data:
+    doapply: false # pkl_skimmed_dec (pklskdec)
+    domergeapply: false # pkl_skimmed_decmerged (pklskdecmerged)
+    docontinueafterstop: false # set to true to resume interrupted processing (existing corrupted output will be overwritten)
+  mc:
+    doapply: false # pkl_skimmed_dec (pklskdec)
+    domergeapply: false # pkl_skimmed_decmerged (pklskdecmerged)
+    docontinueafterstop: false # set to true to resume interrupted processing (existing corrupted output will be overwritten)
+
+analysis:
+  type: YYYY # used unless specified explicitly as do_entire_analysis -a type_ana
+  # Do each period separately including merged (true)
+  # Do only merged (false)
+  doperperiod: false
+  data:
+    histomass: true # processer: process_histomass
+    dojetsystematics: false # analyzer: jetsystematics
+  mc:
+    histomass: true # processer: process_histomass
+    efficiency: false # processer: process_efficiency, process_response, process_unfolding
```

Comment:
What is the benefit of having a separate config file with only histomass enabled? If the point here is to run only the processor part of the analysis in a script, I would advocate for making a default_processor.yml file instead with efficiency enabled as well which makes more sense in terms of running different blocks of the framework separately. You can always change the flags locally to your current needs but the committed changes should be useful in a longer term.

Reply:
This has no sense at all, it was just for me to get a grasp on the code. It will go away together with the [WIP]